Gopkg.lock

```diff
@@ -55,9 +67,15 @@
   packages = ["."]
   revision = "287cf08546ab5e7e37d55a84f7ed3fd1db036de5"
 
+[[projects]]
+  branch = "master"
+  name = "k8s.io/apimachinery"
+  packages = ["pkg/util/yaml"]
+  revision = "a04e753f5223cf882db01ac64212682ea28767b4"
```

Comment:
The dependent PR in Kubernetes has been merged and the new revision should be `3bb012582b9f1e422e5667594c183556669e77ab` here.

Reply:
Thanks - I'll have a look at it and merge it!

Reply:
Thanks!